### PR TITLE
Support for IntelliJ 2022.3.2

### PR DESCRIPTION
**Pull request by ammachado**

### CODE REVIEW

File: build-all.sh [hunk 1]
```diff
@@ -1,6 +1,6 @@
 #!/bin/bash
 
-for v in "2022.1.4" "2022.2.4" "2022.3.1"; do
+for v in "2022.1.4" "2022.2.4" "2022.3.1" "2022.3.2"; do
```
Comment (essobedo) [inline build-all.sh:3]: Only the last bug fix version of each supported minor versions should be tested, so could you please update 2022.3.1

**Comment by essobedo** [comment]
I forgot that with this version, it doesn't compile anymore, it is a known issue https://github.com/camel-tooling/camel-idea-plugin/issues/788

**Comment by ammachado** [comment]
Updated plugin version and JDK, as per this [blog post](https://blog.jetbrains.com/platform/2022/08/intellij-project-migrates-to-java-17/).

**Comment by essobedo** [comment]
I'm afraid that migrating to Java 17 is more complicated than this, there are breaking changes so we will need to drop support of 222 versions.
Have you tried to launch the tests locally with your changes?

**Review by essobedo — APPROVED** [review]
It looks like it works properly, thx for the PR 👍

**Pull request merged**